### DISCUSSION TITLE
Closing open bugs as `Known Issue`

### DISCUSSION THREAD

**Discussion by jonasstein** [audacity/audacity#2887] [Development]
Several bugs were closed as `Known Issue` (https://github.com/audacity/audacity/issues?q=is%3Aissue+label%3A%22Known+Issue%22+is%3Aclosed) . This confuses the users and reporters.
I think the ticket should remain open unless the bug is fixed, because it is still an open bug.

Instead of `Known Issue` I would suggest something like `cannotfix` or `fixlater`.

**Comment by aklapper** [comment]
Semantically, all issues reported here are "known issues" because this issue tracker is public so such acknowledgment doesn't add much apart from confusion?

If you don't plan to track existing unresolved issues in a public issue tracker, then maybe it's more consequent not to have a public issue tracker (though I guess that's not what you're after but rather dealing with issue backlog bankruptcy?).

If you don't plan to fix this issue (which would be a legitimate decision because product management is about saying No), then it would be helpful to also add an explanatory comment that you simply don't plan to fix this or look into this. Thanks!

**Reply by Tantacrul** [reply]
@aklapper - we will happily explain our reasons for closing issues but we won't spend too much time with people who whinge or complain rudely. So, kindly either report issues in a helpful way, or find another group to whinge at.

**Reply by aklapper** [reply]
@Tantacrul: That generally makes sense. However, I don't think I "whinged" - I merely requested that actions are explained so people don't waste time contributing in unhelpful ways. If you think that my original ticket at https://github.com/audacity/audacity/issues/642 was not "helpful" then you are welcome to point out which exact info was missing in that ticket. Thanks for keeping this constructive.

**Comment by LWinterberg** [comment]
One of the problems was that there were multiple bugs with their respective issues which all would get fixed by stuff that's already on our roadmap. All the wxWidgets stuff for example will turn obsolete once we switch to Qt with Audacity 4.0. Rigidly following the procedure Github suggests would mean that eventually there'd be a big Qt PR which resolves all these issues. However, there's no point in doing that as Qt is not about any of that. So by extension, there's no point in keeping the bugs open as they don't serve any purpose to developers. 

Other issues are so far off our roadmap that we won't fix them, so again, keeping them open doesn't serve any purpose. 

All of this is [documented in the wiki](https://github.com/audacity/audacity/wiki/Known-Issues).

Incidentally, both this procedure and the name has been [inspired by Blender](https://code.blender.org/wp-content/uploads/2019/12/flow-chart-ish-6-months.png).

**Comment by Tantacrul** [comment]
I'm going to write a policy for how we triage issues shortly. In many cases - especially when the issues are low-priority quirks in the 3.x UI that have been around for a long while, we don't plan to fix these, since we are creating a new interface. For those types of issues, it doesn't make sense to keep them permanently open. More to come on this later.

I do take the point that a policy needs to be clearly communicated.

**Comment by LWinterberg** [comment]
Update: https://github.com/audacity/audacity/discussions/3048
